content/source/docs/enterprise/install/installer.html.md

```diff
@@ -143,19 +143,19 @@ TFE runs `terraform plan` and `terraform apply` operations in a disposable Docke
 ### Requirements
  - The base image must be `ubuntu:xenial`.
  - The image must exist on the Terraform Enterprise host. It can be added by running `docker pull` from a local registry or any other similar method.
- - CA certificates must be available when Terraform runs. During image creation, a file containing all necessary PEM encoded CA certificates must be placed in `/etc/ssl/certs/ca-certificates.crt`.
- - Terraform must not be installed on the image. TFE will take care of that at runtime.
+ - A file containing all necssary PEM-encoded CA certificates must be placed in `/etc/ssl/certs/ca-certificates.crt`. This includes both custom CA certificates and default CA certificates that come with the operating system. The CA certificates configured in the [CA Bundle settings](#certificate-authority-ca-bundle) will not be automatically added to this image at runtime.
+ - Terraform must not be installed on the image. Terraform Enterprise will take care of that at runtime.
 
  This is a sample `Dockerfile` you can use to start building your own image:
 
  ```
 # This Dockerfile builds the image used for the worker containers.
 FROM ubuntu:xenial
 
-# Inject the ssl certificates
+# Inject the CA certificates.
 ADD ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
```

Comment:
Doesn't that require you to place all the default CA certs AND any custom CA certs in the file `ca-certificates.crt`?

Wouldn't it be more common to trust the same default certs as the original worker image, and then add in any custom certificates?

Reply:
> Doesn't that require you to place all the default CA certs AND any custom CA certs in the file `ca-certificates.crt`?

Yep!

> Wouldn't it be more common to trust the same default certs as the original worker image, and then add in any custom certificates?

Technically speaking, yes. In terms of the application, certain programs are configured to use `/etc/ssl/certs/ca-certificates.crt` as their CA bundle file. I _believe_ we could instruct users to drop their `.crt` files into `/usr/local/share/ca-certificates` and then run `update-ca-certificates` to regenerate `/etc/ssl/certs/ca-certificates.crt`. Problem is, that process is Ubuntu/Debian specific (which is fine right now since that's all we support for an alternative worker image, I'm just thinking for the future). Either way, I agree that we should make this process easier for the user (i.e. users should only need to provide custom CA certificates).